src/lang/base/ScillaLexer.mll

```diff
@@ -140,10 +141,11 @@ and read_string buf =
   | _ { raise (Error ("Illegal string character: " ^ Lexing.lexeme lexbuf)) }
   | eof { raise (Error ("String is not terminated")) }
 
-and comment depth =
+(* Nested comments, keeping a list of where comments open *)
+and comment depth braces =
```

Comment:
`braces` has strictly more information than `depth`. Could we drop `depth` parameter, please?